ragna/deploy/_api/schemas.py

```diff
@@ -75,7 +85,7 @@ class Message(BaseModel):
     content: str
     role: ragna.core.MessageRole
     sources: list[Source] = Field(default_factory=list)
-    timestamp: datetime.datetime = Field(default_factory=datetime.datetime.utcnow)
```

Comment:
`datetime.datetime.utcnow` is deprecated starting with Python 3.12 and thus I refactored this whole thing to be timezone aware.